h/templates/client/annotation.html

```diff
@@ -1,16 +1,16 @@
-<header class="annotation-header" ng-if="!vm.user()">
+<header class="annotation-header" ng-if="::!vm.user()">
```

Comment:
This does change when you create an annotation or highlight while logged out, then log in. That feature seems to still work correctly on this branch though which surprises me a little, but I guess the AnnotationController instance is destroyed and a new one created for it from the drafts service.